engine/schema/src/main/java/com/cloud/network/TungstenGuestNetworkIpAddressVO.java

```diff
@@ -0,0 +1,78 @@
+package com.cloud.network;
```

Comment:
don't forget license headers, the integration builds will fail on ommision.

Reply:
@DaanHoogland regarding your first change that param have been moved in GuestNetworkGuru. It's not there anymore.
Regarding the license i already spoked with Huy and he will add them soon.

Reply:
```suggestion
// Licensed to the Apache Software Foundation (ASF) under one
// or more contributor license agreements.  See the NOTICE file
// distributed with this work for additional information
// regarding copyright ownership.  The ASF licenses this file
// to you under the Apache License, Version 2.0 (the
// "License"); you may not use this file except in compliance
// with the License.  You may obtain a copy of the License at
//
//   http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing,
// software distributed under the License is distributed on an
// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
// KIND, either express or implied.  See the License for the
// specific language governing permissions and limitations
// under the License.
package com.cloud.network;
```